### PR TITLE
SDK-2255 Share V2 - Fetch QR code

### PR DESCRIPTION
**Pull request by michaelyoti**

### CODE REVIEW

File: src/digital_identity_service/share.qr.code.fetch.result.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+   * @param {Object} response
+   */
+  constructor(response) {
+    Validation.isString(response.id, 'QR code ID');
+    this.id = response.id;
+
+    Validation.isString(response.expiry, 'Expiry');
+    const expiryDate = new Date(response.expiry);
+    if (expiryDate.toString() === 'Invalid Date') throw TypeError('Expiry must be a date like string');
+    this.expiry = expiryDate;
```
Comment (abdalmajeed-yoti) [inline src/digital_identity_service/share.qr.code.fetch.result.js:23]: You can use `Validation.isStringDate`.

Comment (laurent-yoti) [reply src/digital_identity_service/share.qr.code.fetch.result.js:23]: Good shout!
Could you make the change @abdalmajeed-yoti  (given M is absent today)

**Review by abdalmajeed-yoti — APPROVED** [review]

**Review by laurent-yoti — APPROVED** [review]
LGTM, please merge.

**Review by laurent-yoti — CHANGES_REQUESTED** [review]
Oops, please don't merge

**Review by laurent-yoti — APPROVED** [review]

**Review by dhivya-pattabhiraman — APPROVED** [review]

**Pull request merged**